flipperdevices/flipperzero-firmware
# [FL-3462] External apps icounter

# What's new

- Give 2 icounter points for app start via the Applications menu

# Verification 

- Open any app via the Applications menu
- Check via logs that the icounter was incremented by 2
- Open any app via the main menu
- Check that icounter wasn't incremented

# Checklist (For Reviewer)
